### DISCUSSION TITLE
How to pass environment variables to GoTrue from supabase CLI?

### DISCUSSION THREAD

**Discussion by ydennisy** [supabase/supabase#5414] [Questions]
Hello!

I would like to enable google auth when running supabase locally for development.

From searching on this forum, I see there are a [bunch of environment variables](https://github.com/supabase/supabase/blob/1ec181222b881b85e5392da5fa5b76b3e63f7465/docker/docker-compose.yml#L21-L45) we can pass.

I have tried `GOTRUE_EXTERNAL_GOOGLE_ENABLED=true supabase start` but this seems to have no effect and I am still receiving `{"code":400,"msg":"Unsupported provider: Provider is not enabled"}` when trying to sign in with Goolge.

Any help appreciated!

**Comment by GaryAustin1** [comment]
Not sure this is help, but here seems to be some work on this:
https://github.com/supabase/cli/pull/138

**Reply by ydennisy** [reply]
Hey @GaryAustin1 yep this would be excellent - would be great to get that merged in if possible :)
